starter_code/models/Movie.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const movieSchema = new Schema({
+  title: String,
+  director: String,
+  stars: Array,
+  image: String,
+  description: String,
+  showtimes: Array,

```

Comment:
Both in this line and above, I would recommend that you dont put just "Array".

Try to be more specific. Ideally you would put [String] to define array of string!